worlds/noita/__init__.py

```diff
@@ -0,0 +1,58 @@
+from BaseClasses import Item, Tutorial
+from worlds.AutoWorld import WebWorld, World
+from . import Events, Items, Locations, Options, Regions, Rules
+
+
+class NoitaWeb(WebWorld):
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up the Noita integration for Archipelago multiworld games.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["Heinermann", "ScipioWright", "DaftBrit"]
+    )]
+    theme = "partyTime"
+    bug_report_page = "https://github.com/DaftBrit/NoitaArchipelago/issues"
+
+
+# Keeping World slim so that it's easier to comprehend
+class NoitaWorld(World):
+    """
+    Noita is a magical action roguelite set in a world where every pixel is physically simulated. Fight, explore, melt,
+    burn, freeze, and evaporate your way through the procedurally generated world using wands you've created yourself.
+    """
+
+    game = "Noita"
+    option_definitions = Options.noita_options
+    topology_present = True
+
+    item_name_to_id = Items.item_name_to_id
+    location_name_to_id = Locations.location_name_to_id
+
+    item_name_groups = Items.item_name_groups
+
+    web = NoitaWeb()
+
+    def get_option(self, name):
+        return getattr(self.multiworld, name)[self.player].value
```

Comment:
Just a small code optimization, since you only use this function in `def fill_slot_data`, you could rewrite remove this method and write `fill_slot_data` like so:

```py
    # Returned items will be sent over to the client
    def fill_slot_data(self):
        return {name: getattr(self.multiworld, name)[self.player].value for name in self.option_definitions}
```